### DISCUSSION TITLE
Can you set a variable to the streaming time?

### DISCUSSION THREAD

**Discussion by DanielLester83** [WarmUpTill/SceneSwitcher#1006] [Q&A] [answered]
Is it possible to set a variable to the total number of seconds that have elapsed since I started streaming? (recording time pauses when you pause the recording)

**Comment by WarmUpTill** [comment, marked as answer]
A build with the requested functionality will be available here in a few minutes:
https://github.com/WarmUpTill/SceneSwitcher/actions/runs/7774946153?pr=1009

![grafik](https://github.com/WarmUpTill/SceneSwitcher/assets/19472752/546839db-cb0b-46b3-963c-0a4acbc7c5e3)

Let me know if you face any issues! :)

**Reply by DanielLester83** [reply]
Thanks seems to be working perfectly.
